apps/ejabberd/src/mod_push_service_mongoosepush.erl

```diff
@@ -44,7 +44,8 @@
 %%--------------------------------------------------------------------
 
 -spec start(Host :: ejabberd:server(), Opts :: list()) -> any().
-start(Host, Opts) ->
+start(_ServerHost, Opts) ->
+    Host = ?MYNAME,
```

Comment:
@michalwski Fixed in #1233 , so Now the change from this PR is not needed / applicable. Therefore, @astro, could you please revert this change after #1233 is merged?